client/account-recovery/controller.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import analytics from 'lib/analytics';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import DocumentHead from 'components/data/document-head';
+import LostPassword from 'account-recovery/lost-password';
+
+const ANALYTICS_PAGE_TITLE = 'Account Recovery';
+
+export default {
+	lostPassword( context ) {
+		const basePath = context.path;
+
+		analytics.pageView.record( basePath, `${ ANALYTICS_PAGE_TITLE } > Lost Password` );
+
+		renderWithReduxStore(
+			<div>
```

Comment:
The [`<Main />` component](https://github.com/Automattic/wp-calypso/tree/master/client/components/main) would be more appropriate to use here in place of `div` for top-level controller element.